frontend/css/homepage.less

```
@@ -348,8 +352,8 @@
           position: absolute;
           left: 0;
           bottom: -10px;
-          width: 100%;
-          border-bottom: 1px solid @gray-light;
+          width: 30%;
+          border-bottom: 1px solid @gray-dark;
```

This was correct before, the comment on figma states this sohuld be the same color as the header-with-line used in other pages which uses gray-light: https://github.com/metabrainz/listenbrainz-server/blob/b368548ec7b52583356ddb3b129a3ebe63bd2fb6/frontend/css/entity-pages.less#L209
```suggestion
          border-bottom: 1px solid @gray-light;
```